modules/dapps.js

```diff
@@ -104,40 +78,9 @@ function DApps (cb, scope) {
 	 * @listens exit
 	 */
 	process.on('exit', function () {
```

Comment:
It makes nothing for now. I have kept it just in case we do something else in the future. @karmacoma do you think we will use it?